controllers/users.js

```diff
@@ -1,37 +1,152 @@
-const jwt = require("jsonwebtoken");
+const jwt = require('jsonwebtoken');
 
 function userDefinedException(message, statusCode) {
   this.message = message;
   this.statusCode = statusCode;
 }
+function getRandomInt(min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+function isStoredTimeWithin10Minutes(otpGeneratedAt) {
+  // Convert stored time to Date object
+  const storedDate = new Date(otpGeneratedAt);
+
+  // Get current time
+  const currentDate = new Date();
+
+  // Calculate difference in milliseconds
+  const differenceInMilliseconds = currentDate - storedDate;
+
+  // Convert difference to minutes
+  const differenceInMinutes = differenceInMilliseconds / (1000 * 60); // 1000 milliseconds in a second, 60 seconds in a minute
+
+  // Check if difference is less than 10 minutes
+  return differenceInMinutes < 10;
+}
+
 function auth(app, sequelize) {
   const { User } = sequelize;
-  app.post("/login", async function login(req, res) {
-    const { username, password } = req.body;
+  app.post('/sendOtp', async function sendOtp(req, res) {
+    const { mobileNumber } = req.body;
     try {
-      if (!username || !password) {
-        throw new userDefinedException(
-          "Please enter both Email and Password",
-          401
-        );
+      if (!mobileNumber) {
+        throw new userDefinedException('Please enter mobile number', 401);
```

Comment:
For some errors are thrown, for some responses are being returned. Make it uniform.